openlibrary/utils/solr.py

```diff
@@ -112,7 +111,7 @@ def get_many(
         return [doc_wrapper(doc) for doc in resp['response']['docs']]
 
     def update_in_place(self, request, commit: bool = False):
-        resp = requests.post(
+        resp = self.session.post(
```

Comment:
Note to self, verify trending work correctly after deploy; that's the spot that makes use of this function.